File: Arcade/Intro/C++/20_arrayMaximalAdjacentDifference.cpp
int arrayMaximalAdjacentDifference(std::vector<int> inputArray) {
    int max=0;
    for(int i=1;i<inputArray.size();i++){
        int difference=abs(inputArray[i-1]-inputArray[i]);
        if(difference>max) max=difference;
    }
    return max;
}

File: Arcade/Intro/JavaScript/20_arrayMaximalAdjacentDifference.js
function arrayMaximalAdjacentDifference(inputArray) {
    
    var max = Math.abs(inputArray[0] - inputArray[1]);
    
    for (var i = 2; i < inputArray.length; i++) {
        var diff = Math.abs(inputArray[i] - inputArray[i-1]);
        max = diff > max ? diff : max;
    } 
    
    return max;
}
